.github/labeler.yml

```diff
@@ -0,0 +1,6 @@
+# Add 'label1' to any changes within 'example' folder or any subfolders
+android:
+- packages/smooth_app/android/**/*
```

Comment:
What about also adding ios and web

Reply:
The android thing was merely an example, but we could get granular, on a file by file basis (possibly line by line, not sure if the syntax allows that)

Reply:
I've added a few line for User management